backends/tests/integration/core/models_tests.py

```diff
@@ -784,6 +784,22 @@ def test_pipeline_fails_job_with_multiple_workers_and_start_condition(self):
     job1.worker_succeeded(task2.name)
     self.assertEqual(pipeline.status, models.Pipeline.STATUS.FAILED)
 
+  def test_pipeline_still_running_with_first_worker_failed_start_condition_success(self):
+    pipeline = models.Pipeline.create(status=models.Pipeline.STATUS.RUNNING)
+    job1 = models.Job.create(pipeline_id=pipeline.id)
+    job2 = models.Job.create(pipeline_id=pipeline.id)
+    models.StartCondition.create(
+        job_id=job2.id,
+        preceding_job_id=job1.id,
+        condition=models.StartCondition.CONDITION.SUCCESS)
+    job1.get_ready()
+    task1 = job1.start()
+    task2 = job1.enqueue(job1.worker_class, {})
+    job1.worker_failed(task1.name)
+    self.assertEqual(pipeline.status, models.Pipeline.STATUS.RUNNING)
```

Comment:
From my point of view, it should not be `RUNNING`. This should be `FAILED`.
Because:
- job2 expect the job1 to succeed
- but job1 had one worker (or more precisely task) which failed

Reply:
worker failed=>job failed
Right, removed the test.